.pre-commit-config.yaml

```
@@ -1,7 +1,7 @@
 repos:
 - repo: https://github.com/Yelp/detect-secrets
-  rev: f6027a0521e044ba46e54611cabd787b7a88d1a9 
+  rev: v1.4.0
   hooks:
     - id: detect-secrets
-      args: ['--baseline', '.secrets.baseline']
-      exclude: package.lock.json
+      args: [ '--baseline', '.secrets.baseline' ]
+      exclude: package.lock.json
```

We've noticed in one of Nat's PRs that the previous `exclude` line had a typo, it should be `package-lock.json`. We might as well change it here given that this particular repository does have this file.

```suggestion
      exclude: package-lock.json
```